OracleDatabase/SingleInstance/extensions/patching/README.md

```diff
@@ -0,0 +1,7 @@
+# PATCH APPLICATION
+
+Applies patches on oracle home. Multiple one-offs can be applied in a single build but only 1 RU can be applied.
+
+Download the release update and one-offs and place them under extensions/patching/patches directory inside subfolders release_update and one_offs respectively.
+
+Once the patches have been placed in the correct directories, use the buildExtensions.sh script to build the extended image with patch support.

```

Comment:
Can we explain how to use this patched image? and how will datafiles get patched?

Reply:
This pull request has not been merged yet. Once it has, it includes documentation on how the image gets patched.

Reply:
I would expect that documentation in README.md as part of this PR itself.

Reply:
It is. There is a `README.md` included in this PR.